app/uk/gov/hmrc/customs/inventorylinking/export/xml/PayloadDecorator.scala

```diff
@@ -38,8 +38,9 @@ class PayloadDecorator() {
         { vpr.authorisedAs match {
             case Csp(badgeIdEoriPair) =>
               <gw:badgeIdentifier>{badgeIdEoriPair.badgeIdentifier.value}</gw:badgeIdentifier>
-                <gw:eori>{badgeIdEoriPair.eori.value}</gw:eori>
-            case NonCsp(eori) => <gw:eori>{eori.value}</gw:eori>
+              <gw:submitter>{badgeIdEoriPair.eori.value}</gw:submitter>
```

Comment:
consider renaming `badgeIdEoriPair` to `badgeIdSubmitterPair`

Reply:
it will be eori at his point in the code, so this is valid name.